Makefile

```diff
@@ -3,13 +3,10 @@ OUT := $(shell pwd)
 .PHONY: default
 default: build test
 
+export TAG ?= latest
 export HUB ?= quay.io/maistra-dev
 export ISTIO_VERSION ?= 1.23.0
-export BUILD_TEST_IMAGE ?= true
-
-ifeq ($(BUILD_TEST_IMAGE),true)
-	export TAG := test
-endif
+export UPLOAD_TEST_IMAGE ?= true
```

Comment:
`UPLOAD_TEST_IMAGE` to me sounds like the image could be pushed to a registry somewhere. How about naming it `USE_LOCAL_IMAGE`, or `USE_REMOTE_IMAGE` with a default of `false`?

Feel free to merge without if you don't think this makes sense.

Reply:
Ok, I changed `UPLOAD_TEST_IMAGE` to `USE_LOCAL_IMAGE`.